lib/util.js

```diff
@@ -1053,3 +1053,7 @@ exports._exceptionWithHostPort = function(err,
   }
   return ex;
 };
+
+// process.versions needs a custom function as some values are lazy-evaluated.
+process.versions[exports.inspect.custom] =
+  (depth) => exports.format(JSON.parse(JSON.stringify(process.versions)));
```

Comment:
Just making sure - should this be "if intlEnabled"? It doesn't need to be, but just a question.